examples/customs info/create_customs_info.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
+
+customs_info = EasyPost::CustomsInfo.create(
+  eel_pfc: 'NOEEI 30.37(a)',
+  customs_certify: true,
+  customs_signer: 'Steve Brule',
+  contents_type: 'merchandise',
+  contents_explanation: '',
+  restriction_type: 'none',
+  restriction_comments: '',
+  non_delivery_option: 'abandon',
+  customs_items: [
+    customs_item, {
```

Comment:
We don't want a variable here and it can be removed.

```suggestion
    {
```